input/pagecontent/terminology.md

```diff
@@ -30,7 +30,9 @@ Many value sets used in this guide are defined in the base FHIR specification, [
 |[Individual Healthcare Identifier Status](https://healthterminologies.gov.au/fhir/ValueSet/ihi-status-1)|[AU Core Patient](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-patient.html)|NCTS|
 |[NameUse](https://hl7.org/fhir/R4/valueset-name-use.html)|[AU Core Patient](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-patient.html)|FHIR|
 |[Order Identifier Type](https://build.fhir.org/ig/hl7au/au-fhir-base//ValueSet-order-identifier-type.html)|[AU eRequesting ServiceRequest](StructureDefinition-au-erequesting-servicerequest.html)|AU Base|
-|[RANZCR Radiology Referral](ValueSet-ranzcr-radiology-referral.html)|[AU eRequesting ServiceRequest](StructureDefinition-au-erequesting-servicerequest.html)|AU eRequesting|
+|[Pathology Procedure](https://healthterminologies.gov.au/fhir/ValueSet/pathology-procedure-1)|[AU eRequesting Pathology Request](StructureDefinition-au-erequesting-servicerequest-path.html)|NCTS|
+|[RCPA - SPIA Requesting Pathology Terminology Reference Set](https://www.healthterminologies.gov.au/integration/R4/fhir/ValueSet/spia-requesting-refset-3)|[AU eRequesting Pathology Request](StructureDefinition-au-erequesting-servicerequest-path.html)|NCTS|
+|[RANZCR Radiology Procedures](ValueSet-ranzcr-radiology-procedures.html)|[AU eRequesting ServiceRequest](StructureDefinition-au-erequesting-servicerequest-imag.html)|AU eRequesting Imaging Request|
```

Comment:
Wrong column (Published) updated. 
Please replace [AU eRequesting ServiceRequest](StructureDefinition-au-erequesting-servicerequest-imag.html) in the 'Profile where used' column with AU eRequesting Imaging Request (hyperlinked to the profile).  
'Published' column should be 'eRequesting'.